docs/index.asciidoc

```diff
@@ -112,6 +113,18 @@ input plugins.
 
 &nbsp;
 
+[id="plugins-{type}s-{plugin}-add_hostname"]
+===== `add_hostname`
+
+  added[5.2.0, Field was added to allow users to not automatically add the `host` field in events.]
```

Comment:
What about "Field was added to allow users to control whether or not the `host` field is automatically added to events."

Reply:
+1 - I like that